src/accuracy_checker/process.py

```diff
@@ -1,7 +1,7 @@
 from result import result
 
 
-class process:
+class Process:
```

Comment:
please rename to something else. Process is a system name.
like 'ProcessHandler' or 'ExternalProcessHandler'